### PR TITLE
Ncl 874

### PR DESCRIPTION
**Pull request by jsenko**

### CODE REVIEW

**Comment by alexcreasy** [comment]
Test this please

File: ui/app/configuration-set-record/_record.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+(function () {
+  /* global _ */
```
Comment (alexcreasy) [inline ui/app/configuration-set-record/_record.js]: Please don't use this JSHint override here. For library functions you can add globals to the project in the .jshintrc  file; there's a globals object you can add it to, it's better practice.

File: ui/app/configuration-set-record/directives/record-directives.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+(function () {
+  /* globals _ */
```
Comment (alexcreasy) [inline ui/app/configuration-set-record/directives/record-directives.js]: Same as before, if you add to the .jshintrc you won't need to add this to every file where you use _.

File: ui/app/index.html [hunk 3]
```diff
@@ -106,8 +107,12 @@
               <li><a href ui-sref="configuration-set.list">Build Configuration Sets</a></li>
             </ul>
           </li>
-          <li ui-sref-active="active">
-            <a ui-sref="record.list" href>Builds</a>
+          <li class="dropdown" ui-sref-active="active">
+            <a href class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Builds<span class="caret"></span></a>
+            <ul class="dropdown-menu" role="menu">
+              <li><a href ui-sref="record.list">Builds Records</a></li>
+              <li><a href ui-sref="configuration-set-record.list">Configuration Set Build Records</a></li>
```
Comment (alexcreasy) [inline ui/app/index.html]: The link text should read "Build Configuration Set Records"

**Comment by jsenko** [comment]
test this please

**Comment by alexcreasy** [comment]
Retest this please

**Comment by alexcreasy** [comment]
If this passes CI after rebase against #497 I'm happy to merge this.

**Comment by jsenko** [comment]
retest this please

**Comment by jsenko** [comment]
retest this please

**Pull request merged**